src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int RANDOM_RANGE = 100;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        switch (figureNumber) {
+            case 0:
+                return isoscelesTrapezoid();
+            case 1:
+                return circle();
+            case 2:
+                return rightTriangle();
+            case 3:
+                return rectangle();
+            default:
+                return square();
+        }
+    }
+
+    private IsoscelesTrapezoid isoscelesTrapezoid() {
+        return new IsoscelesTrapezoid(
+                random.nextInt(RANDOM_RANGE),
+                random.nextInt(RANDOM_RANGE),
+                random.nextInt(RANDOM_RANGE));
```

Comment:
```suggestion
    private IsoscelesTrapezoid isoscelesTrapezoid() {
    	int smallBase = random.nextInt(RANGOM_RANGE);
    	int bigBase = random.nextInt(RANDOM_RANGE);
    	int height = random.nextInt(RANDOM_RANGE);
        return new IsoscelesTrapezoid(smallBase, bigBase, height);
```
same goes to all methods